### PR TITLE
More mappings for net/minecraft/level

### PR DESCRIPTION
**Pull request by PiTheGuy**
This PR adds mappings for net/minecraft/world/level/chunk and net/minecraft/world/level/pathfinder

### CODE REVIEW

File: data/net/minecraft/world/level/chunk/ProtoChunk.mapping [hunk 1]
```diff
@@ -1,15 +1,19 @@
 CLASS net/minecraft/world/level/chunk/ProtoChunk
 	METHOD <init> (Lnet/minecraft/world/level/ChunkPos;Lnet/minecraft/world/level/chunk/UpgradeData;Lnet/minecraft/world/level/LevelHeightAccessor;Lnet/minecraft/core/Registry;Lnet/minecraft/world/level/levelgen/blending/BlendingData;)V
-		ARG 1 pos
+		ARG 1 chunkPos
 		ARG 2 upgradeData
 		ARG 3 levelHeightAccessor
+		ARG 4 biomeRegistry
+		ARG 5 blendingData
 	METHOD <init> (Lnet/minecraft/world/level/ChunkPos;Lnet/minecraft/world/level/chunk/UpgradeData;[Lnet/minecraft/world/level/chunk/LevelChunkSection;Lnet/minecraft/world/ticks/ProtoChunkTicks;Lnet/minecraft/world/ticks/ProtoChunkTicks;Lnet/minecraft/world/level/LevelHeightAccessor;Lnet/minecraft/core/Registry;Lnet/minecraft/world/level/levelgen/blending/BlendingData;)V
-		ARG 1 pos
```
Comment (PiTheGuy) [inline data/net/minecraft/world/level/chunk/ProtoChunk.mapping:7]: This was changed to match the super class.

File: data/net/minecraft/world/level/chunk/UpgradeData.mapping [hunk 2]
```diff
@@ -4,6 +4,9 @@ CLASS net/minecraft/world/level/chunk/UpgradeData
 		ARG 2 level
 	METHOD <init> (Lnet/minecraft/world/level/LevelHeightAccessor;)V
 		ARG 1 level
+	METHOD loadTicks (Lnet/minecraft/nbt/CompoundTag;Ljava/lang/String;Ljava/util/function/Function;Ljava/util/List;)V
+		ARG 0 tag
```
Comment (Lanse505) [inline data/net/minecraft/world/level/chunk/UpgradeData.mapping:8]: ``tag, identifier, valueFunction, ticks``

**Review by Lanse505 — CHANGES_REQUESTED** [review]
Looks good to me minus one change

File: data/net/minecraft/world/level/chunk/UpgradeData.mapping (hunk 2 above)
Comment (PiTheGuy) [reply data/net/minecraft/world/level/chunk/UpgradeData.mapping:8]: Changed

**Review by Lanse505 — APPROVED** [review]

**Pull request merged**